### DISCUSSION TITLE
Update profile photo via fileupload instead of Gravatar

### DISCUSSION THREAD

**Discussion by fzn0x** [npm/feedback#590] [Website] [answered]
according to the title, the ease of changing profile photos will be even more easier without gravatar

**Comment by MylesBorins** [comment, marked as answer]
@fncolon this has been asked previously but unfortunately we still have the same answer as before. We know this experience isn't ideal but we can't commit resources to do the work at this time.

https://github.com/npm/feedback/discussions/461
